## Camera Information
geometry_msgs/PoseStamped camera_pose
float32[16] proj_matrix

# User selection
geometry_msgs/Point32[] vertices # TODO: Make a 2d point type?; Ignore "z"

bool shift
bool control
bool alt
